cmd/image.go

```diff
@@ -323,6 +323,17 @@ func imageDeleteCommandHandler(cmd *cobra.Command, args []string) {
 	if err != nil {
 		exitWithError(err.Error())
 	}
+
+	if c.CloudConfig.Platform == "azure" || provider == "azure" {
+		azure := p.(*api.Azure)
+
+		err = azure.Storage.DeleteFromBucket(c, args[0]+".vhd")
```

Comment:
i know i originally wrote this but the args[0] ".vhd" thing could be defined somewhere

Reply:
sure. I will make necessary changes.